### PR TITLE
feat: Add /help command

### PR DESCRIPTION
**Pull request by mihakrajnc**
## What does this PR change?

Adds a /help command with a small refactor to remove deferred command creation and simplify the code.

fix:#2567

## How to test the changes?

1. Launch the explorer
2. Try executing /help and other commands
3. Check if command descriptions printed in /help are accurate.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by m3taphysics — APPROVED** [review]
Great!

**Review by dalkia — APPROVED** [review]
Congrats on your first PR! 👏 👏 👏

**Review by NickKhalow — APPROVED** [review]

**Comment by aixaCode** [comment]
Would be great if we could add an intro message with info about /help command, like we did in the old client.
Screenshot 2024-12-11 173438
Overall, looks really solid 👏

File: Explorer/Assets/Scripts/Global/Dynamic/ChatCommands/GoToChatCommand.cs [hunk 1]
```diff
@@ -15,10 +15,12 @@ public class GoToChatCommand : IChatCommand
         private const string COMMAND_GOTO_LOCAL = "goto-local";
         private const string PARAMETER_RANDOM = "random";
 
-        public static readonly Regex REGEX =
+        public Regex Regex { get; } =
             new (
                 $@"^/({ChatCommandsUtils.COMMAND_GOTO}|{COMMAND_GOTO_LOCAL})\s+(?:(-?\d+)\s*,\s*(-?\d+)|{PARAMETER_RANDOM})$",
                 RegexOptions.Compiled);
+        public string Description => "/goto - Teleport to a specific location.";
```
[the remark this answers is not in the document]
Comment (mihakrajnc) [reply Explorer/Assets/Scripts/Global/Dynamic/ChatCommands/GoToChatCommand.cs:22]: Good point! I'll add parameters and more info to all of them.

[the remark this answers is not in the document]
Comment (popuz) [reply Explorer/Assets/Scripts/Global/Dynamic/ChatCommands/GoToChatCommand.cs]: with goto there is also `local`  parameter as I remember

[the remark this answers is not in the document]
Comment (aixaCode) [reply Explorer/Assets/Scripts/Global/Dynamic/ChatCommands/GoToChatCommand.cs]: I intentionally skipped the `local`, as officially we don't support worlds with multiple scenes, just to keep it simple for now.

**Pull request merged**